README.md

```diff
@@ -90,35 +92,80 @@ The sweeping function uses [`tune-sklearn`](https://github.com/ray-project/tune-
 The schema for your simulator resides in [`conf/simulator`](./conf/simulator). After defining your states, actions, and configs, you can run the simulator as follows:
 
 ```bash
-python ddm_predictor simulator=$YOUR_SIM_CONFIG.yaml
+python ddm_predictor.py simulator=$YOUR_SIM_CONFIG
 ```
 
+> NOTE: If wanting to train with bonsai, make sure `conf/simulator/policy` is set to "bonsai" instead of "random"
+
 If you would like to test your simulator before connecting to the platform, you can use a random policy:
 
 ```bash
-python ddm_predictor.py simulator=$YOUR_SIM_CONFIG.yaml simulator.policy=random
+python ddm_predictor.py simulator=$YOUR_SIM_CONFIG simulator.policy=random
 ```
 
-## Build Simulator Package
+> NOTE: The optional flags should NOT have .yml; it should just be the name of the config file
+
+If you're having trouble running locally, chances are you need to set up your workspace and access key configs. You can do this by using environment variables or the following command
 
 ```bash
-az acr build --image <IMAGE_NAME>:<IMAGE_VERSION> --file Dockerfile --registry <ACR_REGISTRY> .
+python ddm_predictor.py --workspace-setup=True
```

Comment:
Should be `python ddm_predictor.py simulator.workspace_setup=True` (which is usually the default)